### PR TITLE
Better cosnole

### LINKED ISSUE #42: Report the result of crates in the report
### Steps to reproduce
1. Run lift

### Expected behavior
I would expect to see the result of each crate.

### Actual behavior
Only the message from each crate is displayed.

### PR DESCRIPTION
**Pull request by steveoh**
closes #42

### CODE REVIEW

File: src/forklift/cli.py [hunk 1]
```diff
@@ -29,13 +29,14 @@
 
 log = logging.getLogger('forklift')
 template = join(abspath(dirname(__file__)), 'report_template.html')
+config_location = join(abspath(dirname(__file__)), '..', 'config.json')
```
Comment (stdavis) [inline src/forklift/cli.py]: 👍 
I'm guessing this will prevent issues when arcpy changes the working directory.

Comment (steveoh) [inline src/forklift/cli.py]: yes.

File: tests/test_cli.py [hunk 2]
```diff
@@ -241,3 +245,18 @@ def test_git_update(self, _validate_repo_mock, _get_repo_mock, clone_from_mock):
 
         clone_from_mock.assert_called_once()
         remote_mock.pull.assert_called_once()
+
+
+class TestReport(unittest.TestCase):
+
+    def test_format_dictionary(self):
+        #: run with --nocapture and look at tox console output
+        good_crate = {'name': 'Good-Crate', 'result': Crate.CREATED, 'crate_message': None}
```
Comment (stdavis) [inline tests/test_cli.py]: `Crate` isn't defined. Need to import it.

Comment (steveoh) [inline tests/test_cli.py]: fixed with https://github.com/agrc/forklift/pull/47/commits/330cdd25624c5587ffb966fef25d80dcaa91ed4d

**Comment by stdavis** [comment]
LGTM

**Pull request merged**